posthog/hogql_queries/experiments/experiment_trends_query_runner.py

```diff
@@ -226,7 +255,63 @@ def calculate(self) -> ExperimentTrendsQueryResponse:
 
         def run(query_runner: TrendsQueryRunner, result_key: str, is_parallel: bool):
             try:
-                result = query_runner.calculate()
+                database = create_hogql_database(team_id=self.team.pk)
```

Comment:
I just want to confirm my understanding: the purpose of this is to create a `Database` instance, which is an object that lets you run HogQL queries across various data sources, like person-events or data warehouse tables.

We do this because we need to build a custom context with our own `Database`, including our own table where we define the join in a custom way. Then, we pass this custom context to the query runner.

Please check if the above is correct and leave a comment :)